nltk/corpus/reader/framenet.py

```diff
@@ -686,6 +686,10 @@ def __getitem__(self, name):
             return v._data()
         return v
 
+    def __lt__(self, other):
+        """ Comparison operator used mainly to sort a list of AttrDict by ID. """
+        return self['ID'] < other['ID']
```

Comment:
I don't think it's guaranteed that `AttrDict` instances have an ID. But that's only an issue if there is code that attempts to compare or sort them, so this is probably fine.

Reply:
Ah now I understand, the `AttrDict` is a generic class. I thought it was a specific object made for Frames.